applications/FluidDynamicsApplication/python_scripts/compute_drag_process.py

```diff
@@ -2,17 +2,18 @@
 import KratosMultiphysics.FluidDynamicsApplication as KratosFluid
```

Comment:
Changes to this file are not related to this PR and should usually be done in a separate PR